app/views/order.py

```diff
@@ -66,11 +72,13 @@ def items_showcase(id, form=None):
         abort(401)
     return render_template('order_items.html', order=order)
 
+
 @order_bp.route('/<id>/edit', methods=['GET', 'POST'])
 @login_required
 def order_edit(id):
     order = Order.query.filter(Order.id == id).first()
-    if current_user.id is not order.courrier_id and not current_user.is_admin():
+    if current_user.id is not order.courrier_id and not current_user.is_admin(
+    ):
```

Comment:
please fix sad code

Reply:
`):`

Reply:
That's because of the yapf formatter which breakes lines after 77 lines.
I would rather fix this by adding black formatting or a config file to the pipeline.

Reply:
+1 for black formatting

Reply:
See my other pr